### DISCUSSION TITLE
Amazon minimum URL option

### DISCUSSION THREAD

**Discussion by wikey** [leon-cleaning-services/leon#160] [Ideas]
Amazon item URLs contain a different structure based on how you found the item. I would love if Leon could remove that metadata by transforming everything to the minimum amazon item URL. 

As an example, if I place this item in my cart and open the link from there, I get this URL:
- https://www.amazon.com/gp/aw/d/B009EEZYE0/ref=ox_sc_act_image_1?smid=ATVPDKIKX0DER&psc=1
but the same item can be accessed directly at:
- https://www.amazon.com/dp/B009EEZYE0/

The second option reveals less about my behavior/interactions with Amazon and is just generally much shorter and cleaner. What does everyone think?

**Comment by svenjacobs** [comment]
Hello @wikey, thanks for the feedback. Leon already has a sanitizer for Amazon product links but it seems that the link from the cart is slightly different. So either the existing sanitizer needs to be adjusted or we need to write a new one for Amazon cart URLs.

**Comment by TPS** [comment]


**Reply by svenjacobs** [reply]
Unfortunately I cannot confirm that this was fixed by PR #180 because the link mentioned @wikey has a different structure

```
https://www.amazon.com/gp/aw/d/B009EEZYE0/ref=ox_sc_act_image_1?smid=ATVPDKIKX0DER&psc=1
```

**Reply by svenjacobs** [reply]
Well, this was fixed with #184 😃

**Comment by svenjacobs** [comment]
This was fixed with #184
